zio-config/src/main/scala/zio/config/actions/Read.scala

```diff
@@ -23,7 +23,7 @@ object Read {
 
             result <- ZIO.fromEither(
                        propertyType
-                         .read(path, value)
+                         .read(path, value.value)
```

Comment:
`value.value` sounds quite confused, maybe we can rename the first `value` to something like `configV`or `configValue`

Reply:
Agree, replaced with `configValue`.